exec/cnex/nstring.c

```diff
@@ -27,6 +27,20 @@ TString *string_createString(size_t length)
     return c;
 }
 
+TString *string_createStringFromData(void *data, size_t len)
+{
+    TString *c = string_newString();
+
+    c->length = len;
+    c->data = malloc(c->length);
+    if (c->data == NULL) {
+        fatal_error("Could not allocate memory (0x%08X bytes) for new data string.", len);
```

Comment:
Why is this printing the number of bytes in hex? That seems unnecessary. Use decimal.

Reply:
Changed to decimal.  (This was originally hex to help stiff out errors that could have been easily located if this error was thrown.  0xFDFDFDFD for example.

Reply:
You changed some other function to decimal, which is fine, but this one needs to be changed too.